core/templates/core/email/resource_request_approved.html

```diff
@@ -7,8 +7,6 @@
 
 Your resource request for {{request}} has been approved!
 
-Reason: {{reason}}
-
```

Comment:
This email really didn't make sense before, and most staff weren't aware of what they were sending.